cobol-parser/src/main/scala/za/co/absa/cobrix/cobol/parser/antlr/ANTLRParser.scala

```diff
@@ -74,6 +74,15 @@ object ANTLRParser {
     visitor.ast
   }
 
+  /**
+    * Replace special characters that have semantics of spaces.
+    */
+  private def filterSpecialCharacters(copybook: String): String = {
+    copybook
+      .replace('\u00a0', ' ') // Non-breakable spaces
+      .replace(0x09.toByte.toChar, ' ') // Tabs
```

Comment:
just curious why use 2 different notation to specify these?

Reply:
Yeah, seems confusing. Will use the same notation.